apps/web/app/dashboard/_components/EmptyState.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { TextBalloonIcon } from "@ui/public";
+import Button from "@ui/src/components/common/Button";
+import { useRouter } from "next/navigation";
+import { memo } from "react";
+
+interface EmptyStateProps {
+  isMeeting: boolean;
+}
+
+function EmptyState({ isMeeting }: EmptyStateProps): JSX.Element {
+  const router = useRouter();
+
+  const handleOnClick = (): void => {
+    router.push(isMeeting ? "/meetings" : "/equipments");
+  };
+
+  return (
+    <div className="h-253 rounded-16 bg-gray-10 flex w-full flex-col items-center py-52">
+      <span className="rounded-12 mb-16 block size-44 bg-gray-200/10 p-10">
+        <TextBalloonIcon className="size-22" />
+      </span>
+      <p className="text-custom-black/80 text-15 mb-24">
+        {isMeeting ? "오늘 예정된 미팅이 없어요." : "대여 예정인 장비가 없어요."}
+      </p>
+      <Button onClick={handleOnClick} variant="Secondary">
+        {isMeeting ? "미팅 잡기" : "장비 대여 신청하기"}
+      </Button>
```

Comment:
```suggestion
      <Button as="Link" href={ROUTES_CONSTANTS} variant="Secondary">
        {isMeeting ? "미팅 잡기" : "장비 대여 신청하기"}
      </Button>
```
아마 as Link태그로 사용하면 handleClick 함수 없이 네비게이션 역할을 해주는 컴포넌트로 사용 가능할듯해한데, 사실 나도 적용해본적은 없어 ...ㅎㅎ
그래도 혹시 가능하다면 페이지 이동에서는 Link태그가 더 우수할거야!

![대충 Link태그 관한 내용](https://velog.io/@eunsun23/Next.js-%EB%9D%BC%EC%9A%B0%ED%8C%85-Link%ED%83%9C%EA%B7%B8-%EB%9D%BC%EC%9A%B0%ED%8C%85%EA%B3%BC-%EA%B4%80%EB%A0%A8%EB%90%9C-%EC%84%B1%EB%8A%A5-%EC%B5%9C%EC%A0%81%ED%99%94)

Reply:
오ㅎㅎ 링크태그로 되는지 확인해봐야겠네 땡큐~~ 저거 라우트 주소는 상수화 해도 괜찮을 것 같아서 방금 뺐어. 이름 PAGE_NAME으로 했는데 바꾸는게 좋을까?

Reply:
ROUTES 아니면 PAGE_NAME중에서 편한대로 하면 될거같아! 
gpt는 이런식으로 답변줬어
```ts
export const ROUTES = {
  DASHBOARD: "/dashboard",
  MEETINGS: "/meetings",
  SEATS: "/seats",
  EQUIPMENTS: "/equipments",

  ADMIN_MEMBERS: "/admin/members",
  ADMIN_TEAMS: "/admin/teams",
  ADMIN_MEETINGS: "/admin/meetings",
  ADMIN_SEATS: "/admin/seats",
  ADMIN_EQUIPMENTS: "/admin/equipments",
};
```